networkit/test/test_structures.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import unittest
+import os
+
+import networkit as nk
+
+class TestStructures(unittest.TestCase):
+
+	def setUp(self):
+		self.G = nk.readGraph("input/jazz.graph", nk.Format.METIS)
+		self.G.indexEdges()
+
+	def testCoverBounds(self):
+		co = nk.Cover(4)
+		co.extend() #cover of size 5
+		co.setUpperBound(4)
+		self.assertEqual(co.upperBound(), 4)
+		self.assertEqual(co.lowerBound(), 0)
+
+	def testCoverSubsets(self):
+		co = nk.Cover(3)
+		co.setUpperBound(3)
+		co.addToSubset(0,0)
+		co.addToSubset(1,1)
+		co.addToSubset(2,2)
+		self.assertEqual(co.numberOfElements(), 3)
+		self.assertEqual(co.numberOfSubsets(), 3)
+		co.removeFromSubset(1,1)
+		self.assertFalse(co.contains(1))
+		co.mergeSubsets(0,1) #0+1 -> 3
```

Comment:
Split the tests for `mergeSubsets`, `removeFromSubset` and general subset tests (like the number).